Main Cleanup of Muon Analysis release notes

sf1919 opened the pull request:
**Description of work.**

The main refactoring the the Muon Analysis release notes ahead of release of 6.2. Includes:-
- Adding links
- Updating titles and headings
- Re-ordering information
- Adding screenshots
- Spell checking

**To test:**
- Check that the links work
- Are the images visible and clearly illustrate the new feature/improvement?
- Grammar/Spelling check


*There is no associated issue.*


---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

sf1919 committed:
Added missing links and tidied titles and headings

sf1919 committed:
Major changes

Added screenshots, reorganised as some content under wrong headings, missing content added, spellcheck.

robertapplin commented on docs/source/release/v6.2.0/muon.rst:
```diff
@@ -11,54 +11,53 @@ Frequency Domain Analysis
 
-Bug Fixes
-#########
+.. image::  ../../images/maxent_dual_plot.PNG
```
I don't think this image is being found. Try using lowercase `.png`

sf1919 replied:
Can you see the other images in the notes? They both have .PNG rather than lowercase.

robertapplin replied:
Yes I can see the other two. Its because the other two have the extension `.PNG` but the new image has it lowercase as `.png`

sf1919 committed:
Additional revisions

Added final set of screenshots, updated text for continuity and added additional links where appropriate.

robertapplin commented on docs/source/release/v6.2.0/muon.rst:
```diff
@@ -11,54 +11,51 @@ Frequency Domain Analysis
-- Can now plot the raw count data in the GUI.
-- It is now possible to perform an Automatic or Manual background correction in the new :ref:`Corrections Tab <muon_corrections_tab-ref>`.
+- It is now possible to exclude a single range from a fit range when doing a fit on the :ref:`Fitting Tab <muon_fitting_tab-ref>`.
+- Added a 'Covariance Matrix' button to the :ref:`Fitting Tab <muon_fitting_tab-ref>` that can be used to open and inspect the normalised covariance parameters of a fit.
```
Can you change the single quotes to be \`\`Covariance Matrix\`\` instead. That is my fault for using the wrong thing previously

robertapplin commented on docs/source/release/v6.2.0/muon.rst:
```diff
@@ -71,28 +68,41 @@ Improvements
 - The plotting now has autoscale active by default.
 - It is now possible to load nexusV2 files in the GUI.
-- Added a table to store phasequads in the phase tab, phasequads also no longer automatically delete themselves
+- Added a table to store phasequads in the phase tab. Phasequads also no longer automatically delete themselves
```
Can we rephrase the last sentence to 'Also, phasequads no longer delete themselves automatically.'

robertapplin commented on docs/source/release/v6.2.0/muon.rst:
```diff
@@ -71,28 +68,41 @@ Improvements
-- Frequency domain analysis can now use groups in :ref:`MuonMaxent <algm-MuonMaxent>` calculations.
 - The labels on the tabs in the GUIs will now show in full
+- When running the :ref:`DynamicKobuToyabe <func-DynamicKuboToyabe>` fitting function you should now be able to see the BinWidth to 3 decimal places.
+- It is now possible to select the normalisation (``analysis_asymmetry_norm``) and group (``analysis_group``) in the results table tab.
```
The end of this sentence should be `... in the Results Tab.`

robertapplin commented on docs/source/release/v6.2.0/muon.rst:
```diff
@@ -71,28 +68,41 @@ Improvements
+  in the range is smaller than the final digit of the first number in the range (e.g. 6018-3 can be used for the range 6018-6023).
+- In the :ref:`Fitting Tab <muon_fitting_tab-ref>` the ``fit to raw`` checkbox can no longer be unchecked if no rebinned data is present.
+- A bug has been fixed in the BinWidth for the :ref:`DynamicKobuToyabe <func-DynamicKuboToyabe>` Fitting Function which caused a crash and did not provide
+  any information about why the value was invalid. Will now revert to last viable BinWidth used and explain why.
```
`It will now revert...`

robertapplin commented on docs/source/release/v6.2.0/muon.rst:
```diff
@@ -71,28 +68,41 @@ Improvements
+- In the :ref:`Fitting Tab <muon_fitting_tab-ref>` the ``fit to raw`` checkbox can no longer be unchecked if no rebinned data is present.
+- A bug has been fixed in the BinWidth for the :ref:`DynamicKobuToyabe <func-DynamicKuboToyabe>` Fitting Function which caused a crash and did not provide
+  any information about why the value was invalid. Will now revert to last viable BinWidth used and explain why.
+- The autoscale option when ``All`` is selected will now show the largest and smallest y value from the all of the plots.
```
I think there is a typo in this sentence

robertapplin reviewed (changes requested):
Nice changes, especially with regards to adding pictures and links to the relevant tabs. I will test the links are working tomorrow. I had a few comments in the meantime

sf1919 committed:
Changes made after review

robertapplin reviewed (approved):
Thanks for the changes, it looks good to go
